demisto_sdk/common/constants.py

```diff
@@ -688,16 +687,16 @@ class PB_Status:
 
 EXTERNAL_PR_REGEX = r'^pull/(\d+)$'
 
-SCHEMA_TO_VALIDATOR = {
-    'integration': REPORT_REGEX,
```

Comment:
Do you know why this was using `REPORT_REGEX`? Was this simply a mistake that slipped through?

Reply:
Seems like a mistake, since in the other declaration it was the correct one.